### ISSUE TITLE
Fix the CI report

### ISSUE THREAD

**Issue by EmileRolley** [datagir/nosgestesclimat-site#825]
Currently, the CI report is broken.

**Comment by Clemog** [comment]
Yep, it's broken for site, I think it's my changes for model side that broke CI here

**Comment by Clemog** [comment]
Should be fixed with https://github.com/datagir/nosgestesclimat-site/commit/158a6fa5649b86fdd10abe827e03415c42ae52b5, I let this issue open until it's checked
